CONTRIBUTING.md

```diff
@@ -245,17 +245,11 @@ ICS adheres to the [trunk based development branching model](https://trunkbasedd
 
 ### Semantic Versioning
 
-ICS uses a variation of [semantic versioning](https://semver.org/).
+ICS follows [semantic versioning](https://semver.org), but with the following deviations (similar to [IBC-Go](https://github.com/cosmos/ibc-go/blob/main/RELEASES.md)):
 
-Note that state breaking changes are a subset of consensus breaking changes. Therefore we'll only refer to the latter when talking about versioning.
-
-ICS is a distributed, IBC based protocol in which multiple blockchains could be affected by a version bump. Therefore incrementing a MAJOR version number indicates that the PR updates, or is a breaking change to the way that the provider and consumer(s) communicate with one another over IBC. If a PR is consensus breaking to both the provider and consumer(s), then it requires a MAJOR version bump.
-
-Incrementing a MINOR version number indicates that a PR is only consensus breaking to the provider, or only to the consumers, where IBC communication remains unchanged.
-
-Incrementing a PATCH version number indicates that a PR is not consensus breaking to the provider or consumers. This could include node API changes, or other miscellaneous and often rare changes.
-
-Pure documentation, testing, and refactoring PRs do not require a version bump.
```

Comment:
Should we keep this, just to clarify?

Reply:
I'm not sure that I agree with the sentence:
> Pure documentation

we should have versioned docs

> testing

We shouldn't release code without testing. If we improved just the testing, we can cut a new patch release with the code better tested or just don't release anything yet. Not every PR needs a release. 

> refactoring 

Same as for testing.